From tonyj@suse.de Tue Mar  4 13:01:20 2008
From: Tony Jones <tonyj@suse.de>
Subject: USB: remove incorrect struct class_device from the printer gadget
Date: Fri, 22 Feb 2008 00:13:36 +0100
Message-Id: <1203635616.2702.8.camel@lov.site>
Mime-Version: 1.0
Content-Transfer-Encoding: 7bit
Status: RO
Content-Length: 348900

This field does nothing, and should not be allowed to stick around
incase someone gets any other ideas...

Signed-off-by: Tony Jones <tonyj@suse.de>
Signed-off-by: Kay Sievers <kay.sievers@vrfy.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/printer.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/gadget/printer.c
+++ b/drivers/usb/gadget/printer.c
@@ -92,7 +92,6 @@ struct printer_dev {
 	u8			*current_rx_buf;
 	u8			printer_status;
 	u8			reset_printer;
-	struct class_device	*printer_class_dev;
 	struct cdev		printer_cdev;
 	struct device		*pdev;
 	u8			printer_cdev_open;
